File: core/worker/analysis/infrastructure/adapters/testdata/fixture-csharp/Controllers/UserController.cs
using Microsoft.AspNetCore.Mvc;
using Acme.Services;
using Acme.Models;
using Sys = System.Text;

namespace Acme.Controllers;

[ApiController]
[Route("api/users")]
public class UserController : ControllerBase
{
    private readonly UserService service;

    public UserController(UserService service)
    {
        this.service = service;
    }

    [HttpGet("{id}")]
    public User GetUser(int id) => service.GetUser(id);

    [HttpPost]
    public void CreateUser(User user) => service.Register(user);
}

File: core/worker/analysis/infrastructure/adapters/testdata/fixture-java/src/main/java/com/acme/web/UserController.java
package com.acme.web;

import com.acme.model.User;
import com.acme.service.UserService;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/users")
public class UserController {
    private final UserService service;

    public UserController(UserService service) {
        this.service = service;
    }

    @GetMapping("/{id}")
    public User getUser(@PathVariable int id) {
        return service.getUser(id);
    }

    @PostMapping
    public void createUser(User user) {
        service.register(user);
    }
}
